hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -1124,6 +1124,43 @@
         },
       ]
     },
+    { name: "FW_OV_SHA3_START",
+      desc: "Firmware override sha3 block start control register",
+      swaccess: "rw",
+      hwaccess: "hro",
+      tags: [// Exclude from writes to these field because they cause side affects.
+             "excl:CsrAllTests:CsrExclAll"]
+      fields: [
+        { bits: "3:0",
+          name: "FW_OV_INSERT_START",
+          desc: '''
+                Setting this field to 0xA will instruct the ENTROPY_SRC main state machine
+                to start the SHA3 process and be ready to accept entropy data. This field should
+                be set prior to writting the FW_OV_WR_DATA register. Once all data has been written,
+                this field should be set to 0x5. Once that happened, the SHA3 block will finish
+                processing and push the result into the ESFINAL FIFO.
+                '''
+          resval: "0x5"
+        },
+      ]
+    },
+    { name:     "FW_OV_WR_STATUS",
+      desc:     "Firmware override FIFO write full status register",
+      swaccess: "ro",
+      hwaccess: "hwo",
+      hwext: "true",
+      tags: [// Internal HW can modify status register
+             "excl:CsrAllTests:CsrExclCheck"]
+      fields: [
+        { bits: "0",
+          name: "FW_OV_WR_STATUS",
```

Comment:
Well that would be too clear :). I thought about that once the function was implemented. Since you brought it up, and think that validates the idea that it is worth changing.

Reply:
cryptic specs are the best, i am excellent at writing them. :)